presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/GlobalState.java

```diff
@@ -0,0 +1,15 @@
+package com.globant.equattrocchio.cleanarchitecture;
+
+import android.app.Application;
+
+import io.realm.Realm;
+
+public class GlobalState extends Application{
```

Comment:
Rename with a name more related. For example .  XXXXXXApplication

Reply:
done